packages/components/pie-button/src/index.ts

```diff
@@ -4,38 +4,47 @@ import { customElement, property } from 'lit/decorators.js';
 
 import styles from './button.scss?inline';
 import { validPropertyValues } from './decorators';
-import { BUTTON_TYPE, BUTTON_VARIANT } from './defs';
+import { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT } from './defs';
 
 // Valid values available to consumers
-export { BUTTON_TYPE, BUTTON_VARIANT };
+export { BUTTON_SIZE, BUTTON_TYPE, BUTTON_VARIANT };
 
 @customElement('pie-button')
 export class PieButton extends LitElement {
     /**
-     * The Button type to use
+     * The size of the button. Available sizes are xsmall, small, medium (default) and large.
```

Comment:
NIT: This comment feels like it's more appropriate for the enum itself. Otherwise if the enum changes we have two places to update